### DISCUSSION TITLE
[FIX] Parsnip seed supply

### DISCUSSION THREAD

**Discussion by biscottij** [sunflower-land/sunflower-land#1169] [Ideas]
Dear friends,

I would like to point out a curious exception regarding the parsnip seed supply compared to other crops.
cauliflower  ->  8 hours to grow   -> 80 seeds
parsnip       -> 12 hours to grow  -> 40 seeds (!)
radish         -> 24 hours to grow  -> 40 seeds

Growing parsnips takes half the time of radishes and about twice the time of cauliflowers so we would expect the seed supply to be between those two: 60 seeds. But no, it's only 40 ! Why ?

This blatant injustice is also the sign of a broader disdain towards parsnip that we can unfortunately witness IRL.

For example if I google "parsnip", the question suggestions I get are "Are parsnips just carrots?" or "What are parsnip good for?". Where is the respect ?

Yes we know, our crops are not as cute as those little pink radishes and not as hype as caulis. But the time of parsnip growers is as valuable as other cultivators so we would like to be treated equally by the shopkeeper !

We would like the team and the community to take into consideration the unfairness of the situation and fix it: 60 parsnip seeds please !

**Comment by Demonbane18** [comment]
I agree it should be at least 60 seeds with or without Mysterious parsnip, it's a bit weird  the seed supply is like that for half a radish/wheat time

**Comment by Yougottabekiddingme123** [comment]
> Dear friends,
> 
> I would like to point out a curious exception regarding the parsnip seed supply compared to other crops. cauliflower -> 8 hours to grow -> 80 seeds parsnip -> 12 hours to grow -> 40 seeds (!) radish -> 24 hours to grow -> 40 seeds
> 
> Growing parsnips takes half the time of radishes and about twice the time of cauliflowers so we would expect the seed supply to be between those two: 60 seeds. But no, it's only 40 ! Why ?
> 
> This blatant injustice is also the sign of a broader disdain towards parsnip that we can unfortunately witness IRL.
> 
> For example if I google "parsnip", the question suggestions I get are "Are parsnips just carrots?" or "What are parsnip good for?". Where is the respect ?
> 
> Yes we know, our crops are not as cute as those little pink radishes and not as hype as caulis. But the time of parsnip growers is as valuable as other cultivators so we would like to be treated equally by the shopkeeper !
> 
> We would like the team and the community to take into consideration the unfairness of the situation and fix it: 60 parsnip seeds please !

Says the holder of a Mysterious Parsnip! You could argue for or against this and I am not against your proposal... But most people come here and make proposals looking only at what suits them personally..

**Reply by Aminbtdds** [reply]
It has nothing to do with holding MP🤷‍♂️
Personally i have MP, with 90syncs per month it gives 330ish sfl approx 25$ after withdrawal tax, who cares about 6$ or 9$ syncs fee🤷‍♂️
It is not about holding or not holding certain NFTs, it is about seeking the answer of "why 40 instead of 60?!"

**Reply by biscottij** [reply]
@[Yougottabekiddingme123](https://github.com/Yougottabekiddingme123)
Yes I get your point.
I guess people are always more incentivized to act when they are directly affected.
I agree that this type of contribution is not as valuable as an original contribution with no direct interest.
But I think that is still valuable for the team & community when there are solid points (which is the case I believe in my post)

**Comment by MistaTino** [comment]
I agree with this definetly since Parnsnip 12 hour famring time is perfect for going to bed and work i use it twice a day and need to sync very often, its rather unfair

**Comment by ricard0z10** [comment]
I agree it should be at least 60 seeds with or without Mysterious parsnip

**Comment by veenhdek** [comment]
What if we expand this idea to a whole separate NFT that can be crafted to increase seed supply maybe a seed truck (inspo attached) that could sit next to the shop and add a 25% (actual percentage can be decided later) boost to amount of seed inventory? Beneficial to all and unique from Kuebiko 
![FE5BDC1C-ADAD-44B7-AEC8-51D0A2D1BB6E](https://user-images.githubusercontent.com/79071868/177472034-f9299339-36e1-43a0-8fd9-3ef0ee7eb66b.jpeg)

**Reply by hujikol** [reply]
you should make another discussion on this, beside on tractor NFT, a pickup NFT is lit with its boosting!

**Comment by Yougottabekiddingme123** [comment]
This would directly affect the treasury since the main source of income is
sync fees.

**Reply by veenhdek** [reply]
Good point! If we move forward with this we’d probably do a whole separate discussion on it to make sure we balance the benefit (burn of resources) to the cost (fewer syncs) there are many ways we can go with it too, requiring a consistent resource burn for fuel for the truck to maintain the perk…etc etc. 

I think we address the topic at hand here with just a yes or no on if we think parsnips should be buffed in supply.

**Reply by Demonbane18** [reply]
you got a point but still the supply is a bit weird since the supply should be able to handle a full day full crop rotation, like other high time seeds, only parsnip is 40 even though it is missing 4 seeds, granted you "might" get some seeds through harvesting but there should be some allowances like other seeds

**Reply by biscottij** [reply]
Yes of course that is a good point.

Even if I think this would be very limited if only parsnips supply changes.

My idea was just to point out that parsnip supply is low relatively to others, not to impact the whole supply & sync economy.
I don't know if there is an intention in the game design to disadvantage certain crops like this. If this is intentional why not but event with that I think this is too much compared to raddish & wheat.

**Comment by adamhannigan** [comment]
@biscottij 

The last thing I want in the community is for any sort of Parsnip Discrimination. 😆 

These are a staple crop and a must-have for winter diets! For the purposes of the game they are to be treated as a luxury-crop. In land expansion they will be used for valuable recipes and growing your bumpkin. For this reason we don't want to provide too many of these (we need to keep supply low) ❤️

**Reply by biscottij** [reply]
Ok, then this completely makes sense !
Thanks for taking the time Adam, can't wait for land expansion.
